policy.yaml

```diff
@@ -5,8 +5,17 @@ description: >-
   repository.
 sources:
   - policy:
-      - github.com/enterprise-contract/ec-policies//policy/lib
-      - github.com/enterprise-contract/ec-policies//policy/release
+      # This is likely what is used in Konflux prod, see the EnterpriseContractPolicy record in
```

Comment:
I'd remove "likely" from here. Let's state that this "should" be what is used in Konflux prod.